roadservice/core/models.py

```diff
@@ -233,13 +293,93 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.type = Role.Type.COUNTY_MODERATOR
 
-    # TODO
-    def assign_county_expert(self):
-        pass
+    @property
+    def county(self):
+        return self.region.get_concrete()
 
     def get_concrete(self):
         return self
 
+    def add_service_team(self, speciality, team_members_users):
+        for team_member in team_members_users:
+            if team_member.has_role():
+                return None
```

Comment:
I put this note here, but we should talk about the use of exceptions. I believe this scenario requires an exception. But an `Exception` alone, doesn't mean a thing. We should create some custom exceptions in `exceptions.py`.